modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -112,7 +107,8 @@ const render = (editor: Editor, uiRefs: ReadyUiReferences, rawUiConfig: RenderUi
   Attachment.attachSystemAfter(eTargetNode, mainUi.mothership);
   attachUiMotherships(editor, uiRoot, uiRefs);
 
-  editor.on('PostRender', () => {
+  // TINY-10343: here is used `SkinLoaded` instead of `PostRender` because if the skin load takes too much time it gives some rendering problem
```

Comment:
Maybe add some details on the rendering problems here:

```suggestion
  // TINY-10343: Using `SkinLoaded` instead of `PostRender` because if the skin loading takes too long you run in to rendering problems since things are measured before the CSS is being applied
```

Reply:
done